scripts/deploy_kibana_proxy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+if [[ -n $(cf app logs | grep running) ]] ; then
```

Comment:
The -n test requires that the string be quoted within the test brackets. Using an unquoted string is an unsafe practice.  Examples:
http://www.tldp.org/LDP/abs/html/comparison-ops.html#STRTEST